learn/guides/ballerina-openapi-support.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: ballerina-openapi-support-left-nav-pages-swanlake
+title: Ballerina OpenAPI Support 
+description: Check out how the Ballerina OpenAPI tooling makes it easy for users to start developing a service documented in the OpenAPI contract.
+keywords: ballerina, programming language, openapi, open api, restful api
+permalink: /learn/ballerina-openapi-support/
+active: openapi
+intro: OpenAPI Specification is a specification that creates a RESTFUL contract for APIs, detailing all of its resources and operations in a human and machine-readable format for easy development, discovery, and integration. Ballerina OpenAPI tooling will make it easy for users to start the development of a service documented in an OpenAPI contract in Ballerina by generating Ballerina service and client skeletons. It supports the user to take the code first API design approach by generating an OpenAPI contract for the given service implementation.
+redirect_from:
+  - /learn/how-to-use-openapi-tools/
+  - /learn/how-to-use-openapi-tools
+  - /learn/using-the-openapi-tools/
+  - /learn/using-the-openapi-tools
+  - /swan-lake/learn/using-the-openapi-tools/
+  - /swan-lake/learn/using-the-openapi-tools
+  - /learn/tooling-guide/cli-tools/openapi
+  - /learn/tooling-guide/cli-tools/openapi/
+  - /learn/ballerina-openapi-support/openapi
+  - /learn/cli-documentation/openapi/#using-the-capabilities-of-the-openapi-tools/
+  - /learn/cli-documentation/openapi/#using-the-capabilities-of-the-openapi-tools
+  - /learn/cli-documentation/openapi/#openapi-validator-compiler-plugin/
+  - /learn/cli-documentation/openapi/#openapi-validator-compiler-plugin
+  - /learn/cli-documentation/openapi/
+  - /learn/cli-documentation/openapi
+---
+
```

Comment:
Shall we add a brief paragraph introducing the openAPI concept and its purpose? @hevayo WDYT?

Reply:
Yeah, In here the brief description will be added when the page load on website, which we added in the intro section in line number 08 🙂